config/.ssh/aws_key.pem.example

```diff
@@ -0,0 +1,3 @@
+-----BEGIN RSA PRIVATE KEY-----
+ABCD
```

Comment:
tbh i dont a see a value , specifying a path to the user creating to add his key. 
we can just give the example without doing this.

Reply:
i prefer to use this key in a different place , even though its this path we use in the container, locally i like to use another path.

Reply:
so if i like to do this, maybe others as well

Reply:
2 reasons i added this file.
1) to have created the .ssh folder and checked the same in. 
2) To have the permission of the file preset to 600 - so we can simply rename the file and use the same. 
Without this permission, the test run fails - and asks us to have strict permissions. this is simply to avoid that headache.

Reply:
`If someone chooses to not use the same, they can of course. It is always a good practise to provide a solution-when it is possible, than let people figure out their own solutions.`

Exactly!! thats why i mentioned to add this as an "example" that were it could be in comments....not actually creating the folder....got it ?

Reply:
This has been added as an "example" file. 
I do not see harm in leaving this in there as an option, since there are not actual secrets shared. 
The main points being that the file permissions have been preset and one does not have to mount files in the lima environment. I do not feel there is a good enough reason to remove this file. Please feel free to not use the same.

Reply:
Since we moved all the files into an examples folder - removing this .pem file example from this location.